code/modules/antagonists/_common/antag_spawner.dm

```diff
@@ -157,7 +159,13 @@
 
 	antag_datum = new()
 	antag_datum.send_to_spawnpoint = FALSE
+
+	if(!our_client.prefs.read_preference(/datum/preference/toggle/nuke_ops_species))
+		var/species_type = our_client.prefs.read_preference(/datum/preference/choiced/species)
+		nukie.set_species(species_type)
```

Comment:
This feels backwards or unnecessary. 
`safe_transfer_prefs_to` should already set them to their pref species.